services/pong/test/service.test.js

```diff
@@ -162,10 +169,23 @@ test('check the device ping response', async () => {
   expect(device_pings[1].ms).toBeGreaterThan(0);
   expect(device_pings[1].ms).toBeLessThan(1000);
 
-  expect(device_pings[2].name).toEqual('non-existent-device');
-  expect(device_pings[2].host).toEqual('no-host');
-  expect(device_pings[2].online).toEqual(false);
-  expect(device_pings[2].ms).toEqual(0);
+  expect(device_pings[2].name).toEqual('google');
+  expect(device_pings[2].host).toEqual('google.com');
+  expect(device_pings[2].online).toEqual(true);
+  expect(device_pings[2].ms).toBeGreaterThan(0);
+  expect(device_pings[2].ms).toBeLessThan(1000);
```

Comment:
I would raise this to like `10000` ms or something high as with line 187, since these tests are requiring ping times to an external source which could be a little long depending on the network that the machine running the tests is on. 1 second was used for the other cases since there's no reason it should take that long to ping locally.